include/codeboot.js

```diff
@@ -1031,28 +1037,6 @@ CodeBootVM.prototype.setLargeFont = function (large) {
     }
 };
 
-CodeBootVM.prototype.setShowDrawingWindow = function (show) {
```

Comment:
The methods are used but removed from the prototype?

Reply:
they no longer exist (replaced by `DrawingWindow.prototype.setShow`)